lib/devise/secret_key_finder.rb

```diff
@@ -7,14 +7,12 @@ def initialize(application)
     end
 
     def find
-      if @application.respond_to?(:credentials) && key_exists?(@application.credentials)
-        @application.credentials.secret_key_base
```

Comment:
I removed this because it is redundant. Both `credentials` and `application.secret_key_base` were added in Rails 5.2 and if the key is found in credentials it will be returned.